src/main/java/com/_up/megastore/controllers/interfaces/ICategoryController.java

```diff
@@ -9,17 +9,20 @@
 import org.springframework.web.bind.annotation.*;
 
 
-
 @RequestMapping("/api/v1/categories")
 public interface ICategoryController {
 
     @PostMapping @ResponseStatus(HttpStatus.CREATED)
     CategoryResponse saveCategory(@RequestBody @Valid CreateCategoryRequest createCategoryRequest);
 
+    @DeleteMapping(value = "/{categoryId}") @ResponseStatus(HttpStatus.OK)
+    void deleteCategory(@PathVariable UUID categoryId);
```

Comment:
Al no retornar nada, el tipo de status debería ser un 204 NO_CONTENT, el cual especifica que la petición terminó correctamente pero no se devuelve ningun tipo de información

```suggestion
    @DeleteMapping(value = "/{categoryId}") @ResponseStatus(HttpStatus.NO_CONTENT)
    void deleteCategory(@PathVariable UUID categoryId);
```